cypress/support/commands.ts

```diff
@@ -152,12 +150,11 @@ export const createTestUser = ({
 					password: finalPassword,
 					deleteAfterMinute,
 					isGuestUser,
-					doNotSetUsername,
 				} as IDAPITestUserOptions,
 			})
 			.then((res) => {
 				return cy.wrap({
-					emailAddress: finalEmail,
+					emailAddress: res.body.emailAddress as string,
```

Comment:
this is similar to this PR https://github.com/guardian/gateway/pull/2791

we need to get the email address from the response, as the email address will be augmented with the test user token